supereeg/brain.py

```diff
@@ -130,7 +130,9 @@ def __init__(self, data=None, locs=None, sessions=None, sample_rate=None,
 
         else:
             if isinstance(data, (Nifti, nib.nifti1.Nifti1Image)):
-               data, locs, meta = _nifti_to_brain(data)
+                warnings.simplefilter('ignore')
```

Comment:
can you add a comment here about why the warning comes up and why we're ignoring it?

Reply:
occurs during:
`if mask_file is None:`
     `mask.fit(nifti)`
but I'm not sure why this is occuring

Reply:
Ah-- I meant "add a comment in the code" :)